orchestrator/nodes/get-context/get-context.html

```diff
@@ -18,6 +18,7 @@
     <div class="form-row">
         <label for="node-input-contextName"><i class="fa fa-edit"></i> <span>Context name</span></label>
         <input type="text" id="node-input-contextName" placeholder="Context name" style="width:250px;">
+        <input type="hidden" id="node-input-contextNameType">
```

Comment:
Is this a text field or a comb box with two options?

Reply:
Yes, it can be `msg` or `str`.

Reply:
ok